### PR TITLE
fix(header): update nav toggle label strings to include open/close

### PR DESCRIPTION
**Pull request by kevinbuhmann**
The previous labels did not adequately describe what the buttons will do.

### CODE REVIEW

File: projects/angular/src/utils/i18n/common-strings.default.ts [hunk 1]
```diff
@@ -73,10 +73,10 @@ export const commonStringsDefault: ClrCommonStrings = {
   // Stack View
   stackViewChanged: 'Value changed.',
   // Responsive Nav
-  responsiveNavToggleOpen: 'Navigation menu',
-  responsiveNavToggleClose: 'Navigation menu',
-  responsiveNavOverflowOpen: 'Navigation overflow menu',
-  responsiveNavOverflowClose: 'Navigation overflow menu',
+  responsiveNavToggleOpen: 'Open navigation menu',
+  responsiveNavToggleClose: 'Open navigation menu',
```
Comment (dtsanevmw) [inline projects/angular/src/utils/i18n/common-strings.default.ts:77]: Shouldn't this be like that or?
```suggestion
  responsiveNavToggleClose: 'Close navigation menu',
```

File: projects/angular/src/utils/i18n/common-strings.default.ts [hunk 2]
```diff
@@ -73,10 +73,10 @@ export const commonStringsDefault: ClrCommonStrings = {
...
+  responsiveNavOverflowOpen: 'Open navigation overflow menu',
+  responsiveNavOverflowClose: 'Open navigation overflow menu',
```
Comment (dtsanevmw) [inline projects/angular/src/utils/i18n/common-strings.default.ts:79]: Shouldn't this be like that or?
```suggestion
  responsiveNavOverflowClose: 'Close navigation overflow menu',
```

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
Two suggestions about the close labels

File: projects/angular/src/utils/i18n/common-strings.default.ts (hunk 1 above)
Comment (kevinbuhmann) [reply projects/angular/src/utils/i18n/common-strings.default.ts]: Thanks for catching this.

File: projects/angular/src/utils/i18n/common-strings.default.ts (hunk 2 above)
Comment (kevinbuhmann) [reply projects/angular/src/utils/i18n/common-strings.default.ts]: Thanks for catching this!

**Review by dtsanevmw — APPROVED** [review]
LGTM!

**Pull request merged**